vulture_os/toolkit/api_parser/waf_cloudflare/waf_cloudflare.py

```diff
@@ -133,14 +129,15 @@ def execute(self):
         # Aware UTC datetime
         current_time = timezone.now()
         since = self.frontend.last_api_call or (timezone.now() - timedelta(hours=24))
-
         # Start cannot exceed a time in the past greater than seven days.
         if since < (current_time - timedelta(hours=168)):
-            logger.info(f"[{__parser__}]:execute: Since is older than 168h, resetting-it",
+            logger.info(f"[{__parser__}]:execute: Since is older than 168h, resetting-it to 167h in the past",
                         extra={'frontend': str(self.frontend)})
-            since = current_time - timedelta(hours=168)
-        # Difference between start and end must be not greater than one hour.
+            since = current_time - timedelta(hours=167)
```

Comment:
so... the `since` value will still be able to go past the current_time - 168h value, but will then be moved 1h forward: is that what you really want?